CMakeLists.txt

```diff
@@ -267,3 +268,25 @@ if(PKG_CONFIG_FOUND)
 	configure_file(${PROJECT_SOURCE_DIR}/cmake/openblas.pc.in ${PROJECT_BINARY_DIR}/openblas.pc @ONLY)
 	install (FILES ${PROJECT_BINARY_DIR}/openblas.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig/)
 endif()
+
+
+set(TARGETS_EXPORT_NAME "${CMAKE_PROJECT_NAME}Targets")
+# export targets for find_package config mode
+export(TARGETS ${OpenBLAS_LIBNAME}
+FILE ${CMAKE_BINARY_DIR}/${TARGETS_EXPORT_NAME}.cmake)
+
+install(TARGETS ${OpenBLAS_LIBNAME}
+        EXPORT ${TARGETS_EXPORT_NAME}
+        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
+
+configure_file("cmake/Config.cmake.in"
```

Comment:
Use `configure_package_config_file` here since we need `check_required_components`.

Reply:
Ah, I had that originally then I looked at how Opencl did it and thought this way was simpler for less dependencies.
I'll make that change tomorrow I guess.